docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -122,245 +103,136 @@ RUN yum update -y && yum install -y https://dl.fedoraproject.org/pub/epel/epel-r
 
-WORKDIR ${INTEL_OPENVINO_DIR}/install_dependencies
-RUN ./install_NEO_OCL_driver.sh --no_numa -y --install_driver ${INTEL_OPENCL} && \
-    yum clean all && rm -rf /var/cache/yum && \
-    yum remove -y epel-release
+1. Set up the environment on the host machine to be used for running Docker*. It is required to execute hddldaemon, which is responsible for communication between the HDDL plugin and the board. To learn how to set up the environment (the OpenVINO package or HDDL package must be pre-installed), see [Configuration guide for HDDL device](https://github.com/openvinotoolkit/docker_ci/blob/master/install_guide_vpu_hddl.md) or [Configurations for Intel® Vision Accelerator Design with Intel® Movidius™ VPUs on Linux](configurations-for-vpu-linux.md).
+2. Run `hddldaemon` on the host in a separate terminal session using the following command:
+    ```sh
+    $HDDL_INSTALL_DIR/hddldaemon
+    ```
+    
+## <a name="run-image"></a>Running the Docker Image on Different Devices
+
+### Running the Image on CPU
+
+Run the Docker image with the following command:
+```
+docker run -it --rm <image_name>
 ```
 
-### Run the Docker* Image for GPU
+Note the following things:
 
-To make GPU available in the container, attach the GPU to the container using `--device /dev/dri` option and run the container:
+- Kernel reports the same information for all containers as for native application, for example, CPU, memory information.
+- All instructions that are available to host process available for process in container, including, for example, AVX2, AVX512. No restrictions.
+- Docker does not use virtualization or emulation. The process in Docker is just a regular Linux process, but it is isolated from external world on kernel level. Performance loss is minor.
 
-```sh
-docker run -it --rm --device /dev/dri <image_name>
-```
-> **NOTE**: If your host system is Ubuntu 20, follow the [Configuration Guide for the Intel® Graphics Compute Runtime for OpenCL™ on Ubuntu* 20.04](https://github.com/openvinotoolkit/docker_ci/blob/master/configure_gpu_ubuntu20.md). 
 
-## Use a Docker* Image for Intel® Neural Compute Stick 2
+### Running the Image on GPU
 
-### Configure and Run the Docker* Image for Intel® Neural Compute Stick 2
+> **NOTE**: Only Intel® integrated graphics are supported.
 
-**Known limitations:**
+Note the following things:
 
-- Intel® Neural Compute Stick 2 device changes its VendorID and DeviceID during execution and each time looks for a host system as a brand new device. It means it cannot be mounted as usual.
-- UDEV events are not forwarded to the container by default it does not know about device reconnection.
-- Only one device per host is supported.
+- GPU is not available in the container by default. You must attach it to the container.
+- Kernel driver must be installed on the host.
+- Intel® OpenCL™ Runtime package must be included in the container.
```

Comment:
Please remove this line, the Intel® OpenCL™ Runtime package is already included as part of the Intel® Graphics Compute Runtime for oneAPI Level Zero and OpenCL Driver mentioned in the `Configuring Docker image for GPU` section

Reply:
Done. :)